specs/codecs/dag-cosmos/basic_types.md

```diff
@@ -0,0 +1,39 @@
+# Tendermint and Cosmos Data Structure Basic Types
+
+```ipldsch
+# Uint is a non-negative integer
+type Uint bytes
```

Comment:
should this be an `int`? or is there something special about the representation?

```suggestion
type Uint int
```

Reply:
This should 100% be int, thanks!